development-tools/clinic.rst

```diff
@@ -1495,6 +1495,27 @@ You can still use a self converter, a return converter, and specify
 a *type* argument to the object converter for :c:macro:`METH_O`.
 
 
+How to convert var-positional parameter functions
+-------------------------------------------------
+
+To convert a var-positional parameter function,
+prepending the parameter name with ``*`` ,
+and the parameter should use the ``object`` converter::
+
+   /*[clinic input]
+   var_positional_sample
+
+       foo: int
+       *args: object
+   [clinic start generated code]*/
+
+The implementation function will receive var-positional arguments
+as a tuple, you can either use the tuple directly
+or parsing them into C types using :c:func:`!PyArg_Parse*` functions.
+
+.. versionadded:: 3.11
```

Comment:
Argument Clinic is somewhat an internal tool so I'm not sure we should mention when it was added, or do we?

Reply:
Since this is the devguide @picnixz, I think it is fine to mention or leave out.

Reply:
I'll leave it at aisk's discretion!